src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  Object.keys(data).forEach((key) => {
+    const elementsOfList = document.createElement('li');
+
+    elementsOfList.textContent = key;
+
+    if (typeof data[key] === 'object') {
+      createTree(elementsOfList, data[key]);
```

Comment:
To ensure the nested structure is correctly built, consider appending the newly created list to the current list item (`elementsOfList`) before making the recursive call to `createTree`. This way, the nested list is added to the list item before further nesting occurs.